CoreUtilities/src/au/gov/asd/tac/constellation/utilities/text/SeparatorConstants.java

```diff
@@ -36,4 +36,5 @@ public class SeparatorConstants {
     public static final String ASTERISK = "*";
     public static final String BLANKSPACE = " ";
     public static final String QUOTE = "\"";
+    public static final String FORWARD_SLASH = "/";
```

Comment:
It seems like this constant is unnecessary.
The original code was using `File.pathSeparator`, which was wrong because we're not defining separate paths.
It should work just fine with `File.separator` in each of the places referring to `FORWARD_SLASH`